POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Services/ActivationService.cs

```diff
@@ -21,6 +25,88 @@ internal class ActivationService
 
         private object _lastActivationArgs;
 
+        private void newNotification()
+        {
+            
+            // In a real app, these would be initialized with actual data
+            string title = "Power Toys";
+            string content = "Power Toys is installed";
+            string image = "../Assets/Logo.png";
+            string logo = "../Assets/MiniLogo.png";
+
+            // Construct the visuals of the toast
+            ToastVisual visual = new ToastVisual()
+            {
+                BindingGeneric = new ToastBindingGeneric()
+                {
+                    Children =
+                    {
+                        new AdaptiveText() {
+                            Text = title
+                        },
+
+                        new AdaptiveText()
+                        {
+                            Text = content
+                        },
+
+                        new AdaptiveImage()
+                        {
+                            Source = image
+                        }
+                    },
+
+                    AppLogoOverride = new ToastGenericAppLogo()
+                    {
+                        Source = logo,
+                        HintCrop = ToastGenericAppLogoCrop.Circle
+                    }
+                }
+            };
+
+            // In a real app, these would be initialized with actual data
+            int conversationId = 384928;
+
+            // Construct the actions for the toast (inputs and buttons)
+            ToastActionsCustom actions = new ToastActionsCustom()
+            {
+                Inputs = { },
+                Buttons =
+                {
+                    new ToastButton("Open PowerToys", new QueryString()
+                    {
+                        // Insert query here
+                    }.ToString())
+                    {
+                        // Insert activation type here
+                    },
+                }
+            };
+
+            // Now we can construct the final toast content
+            ToastContent toastContent = new ToastContent()
+            {
+                Visual = visual,
+                Actions = actions,
+
+                // Arguments when the user taps body of toast
+                Launch = new QueryString()
+                {
+                    // Insert query here
+                }.ToString()
+             };
+
+            // And create the toast notification
+            var toast = new ToastNotification(toastContent.GetXml());
+
+            toast.ExpirationTime = DateTime.Now.AddDays(2);
+
+            toast.Tag = "18365";
```

Comment:
what is significant about the constant 18365?

Reply:
Tag is removed in more recent commit